### ISSUE TITLE
initial state/directions mismatch in "Try out some policies" section

### ISSUE THREAD

**Issue by RwwL** [MKorostoff/incarceration-in-real-numbers#3]
First off, THIS PROJECT IS AMAZING, thank you very much for producing it. Incredible work.

In the "Try out some policies" section, the directions say "Tap or mouse over a country to reveal its name" but the country names in the bubbles are visible immediately upon scrolling into the section, and tapping on/mousing over them has no effect.

(latest Chrome desktop on Big Sur)

**Comment by mkorostoff** [comment]
So, the specific bug you're describing doesn't actually exist (it's just a miss-understanding of the instructions) but there is a closely related bug that _does_ exist, which I've known about for a long time and would like to fix. Only about 10 countries have their name displayed in the initial state. The vast majority have no name displayed, and these are the ones you can reveal the name by mousing over.

But! In iOS safari, tapping does not reveal the country name (android works fine). For this reason, I don't show the message "tap a country" on mobile devices, because a lot of mobile devices will simply not have this capability. Ideally, I'd like to 1)  fix tap-to-view in iOS and 2) add the "tap to view" language for mobile devices. Pull requests welcome.

**Comment by RwwL** [comment]
🤦🏼 , not sure how I overlooked all the additional bubbles that don't show names; those big ones with the names take all the attention. Makes sense; maybe the written directions could be slightly clearer but I'm not sure what to suggest without it getting too wordy, so thanks and never mind on that.

Re: the real issue, I'm pretty sure the problem is that iOS won't show hover styles for elements unless they take focus, so adding `tabindex="0"` to any `.country-bubble-small` elements will make them focusable and get it working (worked for me when live-fiddling with the DOM in the iOS Simulator, anyway). I'll see if I can get you a PR later.

**Comment by mkorostoff** [comment]
@RwwL hmm, that makes sense, and without having tested it, I'm 90% sure that approach would work.  Trouble is, it basically breaks the tab key as a useful navigational tool. Like, right now the tab key works in a pretty reasonable way. If you start out at the very top of the page, you can tab your way through all of the links down to the bottom. Tab indexing all the country bubbles would mean a tab user gets stuck on the bubble chart while they are forced to tab through like 150 countries.

One approach that avoids this problem would be to listen for the click/tap event in JS and then add/remove a class to each bubble when tapped, then key off of that class in CSS, instead of keying off the `:hover` pseudoclass. Admittedly, this code would be slightly annoying to write, and I have to wonder if there's an easier way. 

Another approach that might work, and avoids new JS logic, is to make each of these bubbles an anchor tag, but with a tab index of -1, so they're focusable, but not tab-able. Without having tested it, I think there's a 65% chance that would work and a 50% chance it would work without weird side effects.

**Comment by RwwL** [comment]
Yep, totally agree adding 100+ additional tab stops to the document isn't really an accessibility/usability win. That `-1` value on the `tabindex` might just be the trick, though.
